inst/plumber/v1/endpoints.R

```diff
@@ -223,6 +223,7 @@ function(req, res) {
 #* Default is FALSE
 
 function(req, res) {
+  browser()

```

Comment:
Hi @tonyfujs , I think we missed in a `browser()` statement here.

Reply:
Good catch! Thanks @shahronak47